lib/JSONSchemaGenerator.js

```diff
@@ -856,22 +926,11 @@ function getFilterFromRawSchema(pathsMap = {}) {
   }, {})
 }
 
-function propertiesGetValidationCompatibility(collectionDefinition) {
-  let properties = {}
-
-  for (const field of collectionDefinition.fields) {
-    if (!typesToIgnoreInQuerystring.has(field.type) && field.name !== MONGOID) {
-      properties[field.name] = specialTypesValidationCompatibility[field.type]
-        ? ({ ...specialTypesValidationCompatibility[field.type]() })
-        : { type: field.type }
-
-      if (field.description) {
-        properties[field.name].description = field.description
-      }
-    }
-  }
-
-  properties = Object.assign(properties, defaultGetQueryParams, {
+function propertiesGetValidation(validationProperties, isNewSchema) {
+  const properties = {
+    ...validationProperties,
+    ...defaultGetQueryParams,
+    ...mandatoryFieldsWithoutId(isNewSchema),
```

Comment:
I'd probably put `mandatoryFieldsWithoutId` to have consistency with the order we had before.

```suggestion
  const properties = {
    ...mandatoryFieldsWithoutId(isNewSchema),
    ...validationProperties,
    ...defaultGetQueryParams,
```
(and also because I personally am accustomed to have the creatorId, createdAt, etc at the beginning, and then all the other fields)

Reply:
Done, sorry I didn't tell you before